src/v/config/validators.cc

```diff
@@ -67,4 +67,53 @@ validate_connection_rate(const std::vector<ss::sstring>& ips_with_limit) {
     return std::nullopt;
 }
 
+std::optional<std::pair<ss::sstring, int64_t>>
+parse_client_group_byte_rate_quota(const ss::sstring& raw_option) {
+    auto del_pos = raw_option.find(":");
+    if (del_pos == ss::sstring::npos || del_pos == raw_option.size() - 1) {
+        return std::nullopt;
+    }
+
+    auto client_group = raw_option.substr(0, del_pos);
+    auto rate_str = raw_option.substr(del_pos + 1);
+
+    std::pair<ss::sstring, int64_t> ans;
+    ans.first = client_group;
+    try {
+        ans.second = std::stoi(rate_str);
+    } catch (...) {
+        return std::nullopt;
+    }
+    return ans;
```

Comment:
Maybe perform some additional validation like quota must be a non zero positive number

Reply:
added